src/test/java/de/lh/tool/unit/config/security/JwtTokenProviderTest.java

```diff
@@ -1,4 +1,4 @@
-package de.lh.tool.config.security;
+package de.lh.tool.unit.config.security;
```

Comment:
Unit-Tests sollten doch eigentlich immer im selben Package liegen wie der zu testende Code.
Natürlich ist mir klar, warum du hier "unit" eingeführt hast. Würds aber nochmal überdenken.

Reply:
Ich hab jetzt nochmal nachgeschaut, von maven ist sogar der src/it-Ordner extra für Integration Tests vorgesehen. komischerweise muss man den dann mit einem Plugin extra einbinden.